includes/Core/Conversion_Tracking/Conversion_Tracking.php

```diff
@@ -156,6 +157,10 @@ function ( Conversion_Events_Provider $active_provider ) {
 			$gtag_event .= "window._googlesitekit.easyDigitalDownloadsCurrency = '" . edd_get_currency() . "';";
 		}
 
+		if ( Feature_Flags::enabled( 'gtagUserData' ) ) {
+			$gtag_event .= 'window._googlesitekit.gtagUserData = true;';
+		}
+
```

Comment:
We don't need to duplicate this. This has already been implemented as part of WooCommerce integration.